app/frontend/controllers/notes_controller.js

```diff
@@ -226,38 +267,38 @@ class NotesController {
   }
 
   handleNotesClick(event) {
+    var verseNotesBox = event.target.closest('.verse-notes');
+
     if (event.target.nodeName == 'A') {
       // If the user is clicking a link within the note ('a' element)
       // we simply return and let Electron handle the default action
       // (Link will be opened in the default browser)
       return;
     }
-
     event.stopPropagation();
 
-    var verseReferenceId = null;
-    var verseNotesBox = $(event.target).closest('.verse-notes');
-
     // If the notes are not empty we need to ensure that the user actually clicked on the notes content
-    if (!verseNotesBox.hasClass('verse-notes-empty')) {
+    if (false && !verseNotes.classList.contains('verse-notes-empty')) {
```

Comment:
Out of curiosity ... Is the `if (false &&` a common way of implementing a condition? I'm not familiar with that approach. It seems rather indirect from a readability perspective.

Reply:
Sorry, I forgot about it. I decided to skip this condition as clicking on the scroll bar event would trigger scrolling and would not bubble up. Seems like it's working just fine. So I will permanently delete this whole condition case.